docs/HowToAddnewBoard.md

```diff
@@ -1,93 +1,182 @@
 # How to add(port) a new board
 
-There are three kinds of folders, core, chip architecture and board.  
-For example, armv7-r is a core architecture, s5j is a chip architecture and artik05x is a board.  
+To support a new board, you should make changes on three folders, *arch*, *board* and *configs*.  
 
-Changes in *[arch](#arch)*, *[board](#board)* and *[configs](#configs)* folders are mandatory to support a new board.  
+> 1. **arch** folder includes *CPU* and *chip* architectures.  
+> 2. **board** folder includes board support package(BSP).  
+> 3. **configs** folder includes build options and flash operations.
 
-core and chip architecture must be added in *[arch](#arch)*.  
-board must be added in *[board](#board)* and *[configs](#configs)*.  
+## Contents
 
-Adding static library or expanding driver will be provided later.
+> [Folder structure](#folder-structure)  
+> [Architecture](#architecture)  
+> [BSP](#bsp)  
+> [How to include headers](#how-to-include-arch-and-board-header-files)  
+> [Configurations](#configurations)
 
-## Arch
+## Folder structure
 
-Changes in *os/arch* directory for a new board includes core and chip architecture.  
-Currently, TizenRT supports only ARM architectures.  
-But if new board is of another architecture, new core architecture folder must be added.  
+```
+	.
+	|- os .- arch - <cpu-name> .- include .- <cpu-subname>
+	|     |                    |          |
+	|     |                    |          `- <chip-name>
+	|     |                    |
+	|     |                    `- src .- <cpu-subname>
+	|     |                           |
+	|     |                           `- <chip-name>
+	|     `- board - <board-name> .- include
+	|                             |
+	|                             `- src
+	|
+	`- build - configs - <board-name>
+```
 
-chip architecture sources must be added in *os/arch/arm/src/chip-name*.  
-chip architecture headers must be added in *os/arch/arm/include/chip-name*.  
+## Architecture
 
-Example: *os/arch/arm/src/s5j*, *os/arch/arm/include/s5j*  
+This includes *CPU* and *chip* architecture inside.  
 
-Refer *[kconfig](#kconfig)* for adding Kconfig entries
+### CPU
 
-## Board
+```
+	.
+	`- os - arch - <cpu-name> .- include - <cpu-subname>
```

Comment:
Hello @sunghan-chang ,

We also have cpu-family along with cpu-artchitecures.
In patch, os/arch/arm/src/armv7-r
cpu-family : arm 
cpu-architecture : armv7-r